tox.ini

```diff
@@ -3,7 +3,7 @@ skipsdist = True
 skip_missing_interpreters = True
 envlist =
     py3{4,5,6,7,8}-test-{api,sdk,ext-wsgi,ext-http-requests}
-    pypy35-test-{api,sdk,ext-wsgi,ext-http-requests}
+    pypy3-test-{api,sdk,ext-wsgi,ext-http-requests}
```

Comment:
Is this #105? Seems unrelated to this PR.

Reply:
Oh, sorry, I missed #105. Should I create a separate PR to fix that?

Reply:
Hmm, since the pytest change may cause this PR to be discussed longer than that simple bugfix, it might be beneficial. On the other hand, it's not a critical bugfix since the tests were green anyway. Your call 😃

Reply:
Hello @Oberon00,
I've just submitted #133 that contains only this particular fix.